### ISSUE TITLE
Bug: Using 'in' operator override the custom field type

### ISSUE THREAD

**Issue by Abdullah0991** [zebzhao/Angular-QueryBuilder#48]
Hi, I use the following setup for my query builder form (Angular 5 + TypeScript):

```
<query-builder class="margin30" [data]="query" [config]="config">
  <div style="display: inline-block;" *queryInput="let rule; type: 'textarea'">
    <textarea style="width: 250px;height: 75px;" [(ngModel)]="rule.value" placeholder="work1=value,work2=value"></textarea>
  </div>
</query-builder>
```

```
public config: QueryBuilderConfig = {
  fields: {
    name: {name: 'Name', type: 'string', operators: ['=', '!=', 'like', 'not like']},
    description: {name: 'Description', type: 'string', operators: ['=', '!=', 'like', 'not like']},
    gender: {name: 'Gender', type: 'category', options: [], operators: ['in', 'not in']},
    works: {name: 'Works', type: 'textarea', operators: ['in', 'not in']}
  }
};
```

As you see I make the last field 'Works' with type 'textarea', but when use the 'in' or 'not in' operators, the field shows 'listbox' instead of 'textarea' control. How this happens?

**Comment by Abdullah0991** [comment]
Any news @zebzhao !?

**Comment by zebzhao** [comment]
This has been fixed in the latest release 0.3.0.
